src/validators/sysNumTimestamp.ts

```diff
@@ -6,30 +6,30 @@ export const validateSysNumTimestampPathParams = (event: APIGatewayProxyEvent) =
   if (!event.pathParameters?.systemNumber) {
     return addHttpHeaders({
       statusCode: 400,
-      body: 'Missing system number',
+      body: JSON.stringify({ errors: ['Missing system number'] }),
```

Comment:
returning valid json here makes it easier to parse, for in integration tests and on the client, further down the line if necessary.